packages/smooth_app/lib/pages/scan/continuous_scan_page.dart

```diff
@@ -35,31 +35,37 @@ class _ContinuousScanPageState extends State<ContinuousScanPage> {
           final double carouselHeight = constraints.maxHeight /
               1.81; // roughly 55% of the available height
           final double viewFinderBottomOffset = carouselHeight / 2.0;
+
+          final List<Widget> children = getScannerWidgets(
+            context,
+            constraints,
+            _model,
+          );
+
+          //Insert scanner at the right position
+          children.insert(
+            1,
+            SmoothRevealAnimation(
```

Comment:
`1`? I would have imagined the end of the list instead. Looks dangerous: you'll probably be better off inserting it inside `getScannerWidgets`.

Reply:
When on last spot the scanner will be on top and hide the other overlays, and the widget on index 0 is a black fullscreencontainer with a svg in the middle which is displayed while the scanner starts